cypress/e2e/supervision/report/edit-report-due-date.cy.js

```diff
@@ -0,0 +1,42 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withOrder()
+    .withSupervisionLevel()
+    .withOrderStatus()
+});
+
+describe(
+  "Edit due report",
+  {tags: ["@supervision", "@search", "supervision-core"]},
+  () => {
+    it("Successfully edit due date on report in supervision", () => {
+      cy.get("@order").then(({id: orderId}) => {
+        cy.get("@client").then(({id: clientId}) => {
+          cy.visit(
+            `/supervision/#/clients/${clientId}?order=${orderId}`
+          );
+        });
+      });
+      cy.get('.TABS_REPORTS').click();
+      cy.get(':nth-child(1) > report-summary > .report-summary-container > .report-summary-action-panel > .lodge-report-container-parent > .extend-report-due-date-link').click();
```

Comment:
This is a very brittle way to address the element. If any of those DOM elements change, this will break. Can you address the button/link by text instead?

Reply:
done